openwisp_monitoring/check/classes/ping.py

```diff
@@ -178,10 +178,10 @@ def _get_metric(self):
             name=check.name,
             object_id=obj_id,
             content_type=ct,
-            field_name='reachable',
-            key=self.__class__.__name__.lower(),
+            configuration=self.__class__.__name__.lower(),
         )
         metric, created = Metric.objects.get_or_create(**options)
+        metric.full_clean()

```

Comment:
I think it's fine to call _get_or_create

Reply:
Very well then :smiley: